_summaries/ulyanov2016instance_2.md

```diff
@@ -1,19 +1,49 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: ulyanov2016instance
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: x / 10
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+  
+Ulyanov et. al. revist the task of image stylization. Previous works like Gatys et. al. (2016) have introduced methods for style transfer with good results but high computational cost.  Ulyanov et. al. (2016) introduced equivalent feed-forward generator networks, however these networks did not achieve results as good as those propsed in Gatys et. al. In this paper Ulyanov improves his generator architecture through the replacement of batch normalization layers with instance normalization layers and keep them at test time. 
+
+![Image Stylization Ex.](ulyanov2016instance_2a.png)
+
 * How is it realized (technically)?
+
+The authors noted that the contrast of the input image should not be a factor in stylization and that contrast of the stylized image should be similar to the contrast of the original style image. As a result, the authors conclude that the network should discard contrast information from the content image. 
```

Comment:
I think it is good start with an introduction why and how they came about the solution which is  instance normalization.Then go on explain the difference in algorithm between the previous method (batch normalization).